src/android/usb_host/android_uvc.cpp

```diff
@@ -923,6 +924,22 @@ void stream_thread_bulk(usbhost_uvc_stream_context *strctx) {
     LOG_DEBUG("Transfer thread stopped for endpoint address: " << ep);
 };
 
+std::shared_ptr<usb_request> create_request(
+        std::shared_ptr<librealsense::platform::usb_device_usbhost> dev,
+        usb_endpoint_descriptor desc,
+        std::shared_ptr<platform::usb_request_callback> callback)
+{
+    auto rv = std::shared_ptr<usb_request>(usb_request_new(dev->get_handle(), &desc),
+                                                [](usb_request* req){usb_request_free(req);});
+    if(!rv){
+        LOG_ERROR("invalid USB request");
```

Comment:
in case this is a common situation and not an exception then this will pollute the terminal.

Reply:
This is done once per start streaming